frontend/src/views/AuthPage.vue

```diff
@@ -0,0 +1,184 @@
+      }
+
+      await userStore.loginUser(user.token);
+      router.push({ name: 'home' });
+    };
+
+    const registerForm = ref({
+      email: '',
+      password: '',
+      repeatedPassword: '',
+      error: null,
+    });
+
+    const register = async e => {
+      e.preventDefault();
+      registerForm.value.error = null;
+
+      if (registerForm.value.password !== registerForm.value.repeatedPassword) {
+        registerForm.value.error = 'Password and repeated password are not matching';
+        return;
+      }
+
+      const user = await authService.registerUser({
+        email: registerForm.value.email,
+        password: registerForm.value.password,
+      });
+
+      if ('error' in user) {
+        registerForm.value.error = user.error;
+        return;
+      }
+
+      await userStore.loginUser(user.token);
+      router.push({ name: 'home' });
+    };
+
+    return { store: userStore, login, loginForm, register, registerForm };
+  },
+};
+</script>
+
+<template>
+  <div class="form-screen">
+    <img src="../assets/img/timetable_illustration.png" />
+    <div class="form-wrapper" :class="{ toggled: isToggled }">
+      <div class="login-form">
+        <h2>Login</h2>
+        <form @submit.prevent="login">
+          <input v-model="loginForm.email" type="email" placeholder="Email" required />
+          <input v-model="loginForm.password" placeholder="Password" type="password" required />
+          <div v-if="loginForm.error" class="form-error">
+            {{ loginForm.error }}
+          </div>
+          <button type="submit">Login</button>
+          <a @click="isToggled = !isToggled" class="link">Want to register?</a>
+          <a class="link">Forgot your password?</a>
+        </form>
+      </div>
+
+      <div class="registration-form">
+        <h2>Register</h2>
+        <form @submit.prevent="register">
+          <input v-model="registerForm.email" type="email" placeholder="Email" required />
+          <input v-model="registerForm.password" placeholder="Password" type="password" required />
+          <input v-model="registerForm.repeatedPassword" placeholder="Repeat password" type="password" required />
+          <div v-if="registerForm.error" class="form-error">
+            {{ registerForm.error }}
+          </div>
+          <button type="submit">Register</button>
+          <a @click="isToggled = !isToggled" class="link">Want to login?</a>
+          <a class="link">Forgot your password?</a>
+        </form>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+.form-screen {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: calc(100vh - 100px);
+  overflow: hidden;
+}
+
+.form-screen img,
+.form-screen .form-wrapper {
+  width: 50%;
+}
+
+.form-wrapper {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 450px;
+  transition: translate 300ms ease-out;
+}
+
+.form-wrapper.toggled {
+  translate: 0 calc(-100vh - 450px);
```

Comment:
This seems to be an [experimental feature](https://developer.mozilla.org/en-US/docs/Web/CSS/translate) and not available in Chrome ☹️ . If you are not aiming to work on Chrome then you can ignore this but I would advise against that. It is the default browser for the time being.

Reply:
yes, what i meant to use was `transform: translate()` but I typed this from my memory and it worked.